source/_integrations/zwave_me.markdown

```diff
@@ -0,0 +1,54 @@
+---
+title: Z-Wave.Me Z-Way
+description: Instructions on how to integrate Z-Wave with Home Assistant via Z-Wave.Me Z-Way.
+featured: true
```

Comment:
```suggestion
```

Reply:
How to become features?

Reply:
We evaluate the featured integrations once in a while and see if we should change it. It's a separate process from adding new integrations.